cmd/krew/cmd/index.go

```diff
@@ -54,9 +55,26 @@ each configured index in table format.`,
 	},
 }
 
+var indexAddCmd = &cobra.Command{
+	Use:     "add",
+	Short:   "Add a new index",
+	Long:    "Configure a new index to install plugins from.",
+	Example: "kubectl krew index add default " + constants.IndexURI,
+	Args:    cobra.ExactArgs(2),
+	RunE: func(_ *cobra.Command, args []string) error {
+		err := indexoperations.AddIndex(paths.IndexBase(), args[0], args[1])
+		if err != nil {
+			return errors.Wrap(err, "failed to add index")
+		}
+		klog.Infoln("Successfully added index")
+		return nil
```

Comment:
honestly this entire func can be 1 line :)
we'd know the success from the command execution anyway, so no need to log that.

Reply:
I thought that it seemed a little nicer with some explicit validation (similar to index/receipt migration) but am fine either way. I'll remove it.